package.json

```diff
@@ -26,6 +26,7 @@
     "date-fns": "^1.29.0",
     "docz-theme-default": "^0.11.1",
     "immutability-helper": "^2.8.1",
+    "moment": "^2.22.2",
```

Comment:
Moment is quite a heavy library, they have a lighter one that tries to use the browser built-ins, would probably cover our case and keep bundle-size down: https://moment.github.io/luxon/

Reply:
https://github.com/digirati-co-uk/timeliner/pull/105/commits/525fd85f709f47746be00269b2dc6ebf94e50a33